packages/node_modules/@webex/plugin-meetings/src/members/util.js

```diff
@@ -156,4 +160,59 @@ MembersUtil.getTransferHostToMemberRequestParams = (options) => {
   };
 };
 
+MembersUtil.genderateSendDTMFOptions = (url, tones, memberId, locusUrl) => ({
```

Comment:
Can you explain the value of this function? It seems that it just puts what it takes into an object

Reply:
Ok, it's better to keep the consistent